events/models.py

```diff
@@ -93,6 +93,18 @@ def published(self):
     def unpublished(self):
         return self.filter(published_at__isnull=True)
 
+    def available(self, child):
+        child_enrolled_event_groups = EventGroup.objects.filter(
+            events__occurrences__in=child.occurrences.all()
+        )
+        return (
```

Comment:
Would be great if there is a comment here 😄

Reply:
I completely agree, added one 🙂